src/config/dynamic-theme-fixes.config

```diff
@@ -2399,21 +2403,6 @@ INVERT
 .as-localnav-browseall-list-title
 .as-localnav-browseall-listitem
 
-CSS
-#globalnav {
-    --globalnav-background: var(--darkreader-neutral-background) !important;
-}
-
-================================
-
-apps.apple.com
-support.apple.com
```

Comment:
You miss this alias after your changes which breaks the navbar!
![obraz](https://github.com/user-attachments/assets/e4ea6caf-3e8b-4c02-8c43-0b9245b6fe36)

I think the easiest way to fix this is to add alias under `apple.com` and if we want to be sure we won't miss anything with prefix on the apple.com then I would add `*.apple.com`

I will add the other comment in the place I mention.